assets/src/settings-page/plugin-suppression.js

```diff
@@ -158,7 +158,16 @@ function ValidationErrorDetails( { errors } ) {
 							) }>
 							<WrapperElement>
 								<a href={ error.edit_url } target="_blank" rel="noreferrer" title={ error.tooltip }>
-									<span dangerouslySetInnerHTML={ { __html: error.title } } />
+									<span>
+										{
+											createInterpolateElement(
+												error.title,
+												{
+													code: <code />,
+												},
+											)
+										}
+									</span>
```

Comment:
Very nice.

Reply:
I suppose if the `error.title` has no `<code>` then it just won't do anything?

Reply:
Yes, If there is no `<code>` in `error.title` it won't do anything.